src/gsi/read_seviri.f90

```diff
@@ -528,8 +528,10 @@ subroutine read_seviri(mype,val_sev,ithin,rmesh,jsatid,&
   call closbf(lnbufr)
   close(lnbufr)
 
+  number_profiles = count(nrec(:) /= 999999,dim=1)
+
   call combine_radobs(mype_sub,mype_root,npe_sub,mpi_comm_sub,&
-     nele,itxmax,nread,ndata,data_all,score_crit,nrec)
+     nele,itxmax,nread,nread,ndata,data_all,score_crit,nrec)
```

Comment:
Should the second `nread` be replaced with `number_profiles`?  Other read routines have `nread,number_profiles` in the call to `combine_radobs`

Reply:
I'm not sure what happened.  These changes were were combined with the format changes.  I see the format changes and the number_profiles initialization and declaration are in the commit but not the change in combine_radobs.  I am going to wait on resubmitting this change until I resolve @DavidHuber-NOAA comments.  It is possible number_profiles is not needed.

Reply:
@RussTreadon-NOAA   the number_profiles variable has been removed from the various read routines and combine_radobs subroutine.  In this case, the second nread was removed.